src/styles/blocks/catalog.scss

```diff
@@ -7,19 +7,24 @@
   padding: 50px 0;
 
   &__card {
-    display: flex;
+    @extend %display-element-properties;
     justify-content: center;
-    align-items: center;
-    box-sizing: border-box;
 
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $animation-duration;
```

Comment:
You change color and transform on hover so you need to specify this properties is transition rule
```suggestion
    transition: transform $animation-duration, color $animation-duration;
```

Reply:
![image](https://user-images.githubusercontent.com/126285904/230786661-55492ec6-2fd8-46b2-b28b-54a65b1bef12.png)
Understand. When operator has two or more properties - it can set different time duration on hover
Got it, will use